Msg_brd - Final for Sprint 2/dbcon.js

```diff
@@ -0,0 +1,9 @@
+var mysql = require('mysql');
+var pool = mysql.createPool({
+    connectionLimit: 10,
+    host: 'classmysql.engr.oregonstate.edu',
+    user: 'cs340_vanruisg',
+    password: '1814',

```

Comment:
Don't forget to set up this via the included instructions using "dotenv". Ideally, we don't want to expose DB credentials in the repo since anyone can explore our git history and find passwords.

Reply:
I've never used dotenv before so I will look in to this